tests/execution_spec.rs

```diff
@@ -454,7 +456,7 @@ impl ExecutionSpec {
             http2_only,
             file: Arc::new(MockFileSystem::new(self.clone())),
             env: Arc::new(Env::init(env)),
-            cache: Arc::new(init_in_memory_cache()),
+            cache: rt.cache,
```

Comment:
we are creating a runtime and then overriding it. I think we can drop the dependency on init_runtime.